addons/website_slides/controllers/main.py

```diff
@@ -183,6 +183,16 @@ def channel(self, channel, category=None, tag=None, page=1, slide_type=None, sor
 
         return request.render('website_slides.home', values)
 
+    @http.route(['/slides/channel/add'], type='http', auth='user', methods=['POST'], website=True)
+    def slide_channel_create(self, *args, **kw):
+        channel = request.env['slide.channel'].create({
+            'name': kw['name'],
+            'description': kw.get('description'),
+            'channel_type': kw.get('course_type', 'training'),
```

Comment:
You receive channel_type_1 / channel_type_2 if I am not wrong ? Course_type does not seem to be given by JS ?

Reply:
We get a dict with `channel_type` (and not `course_type`), but the value a are correct.